api/v1beta2/common.go

```diff
@@ -19,38 +19,29 @@ package v1beta2
 import (
 	"strconv"
 
+	"k8s.io/apimachinery/pkg/util/intstr"
 	"k8s.io/apimachinery/pkg/util/validation/field"
 )
 
 func defaultIBMPowerVSMachineSpec(spec *IBMPowerVSMachineSpec) {
-	if spec.Memory == "" {
-		spec.Memory = "4"
+	if spec.MemoryGiB == 0 {
+		spec.MemoryGiB = 4
 	}
-	if spec.Processors == "" {
-		spec.Processors = "0.25"
+	if spec.Processors.StrVal == "" && spec.Processors.IntVal == 0 {
+		spec.Processors = intstr.FromString("0.5")
```

Comment:
the default we can still keep 0.2 for the capibm project.

Reply:
Fixed.